tests/modeladmin/tests.py

```diff
@@ -533,6 +534,34 @@ class BandAdmin(ModelAdmin):
             ['extra', 'transport', 'id', 'DELETE', 'main_band']
         )
 
+    # custom log actions behaviour ####################################
+    def test_custom_log_actions(self):
+        class BandAdmin(ModelAdmin):
+            def log_addition(self, request, object, message):
```

Comment:
I think it would be enough to test that the default implementations return the `LogEntry`. You're indirectly testing it now by calling `super()` but the custom methods don't seem necessary.